WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/RenameAttributesViewController.swift

```diff
@@ -81,9 +85,10 @@ private extension RenameAttributesViewController {
 
 extension RenameAttributesViewController {
     @objc private func doneButtonTapped() {
-        // TODO: Update the attribute name if it has changed
-        // For now, just navigate back
-        navigationController?.popViewController(animated: true)
+        guard let newAttributeName = viewModel.newAttributeName else {
+            return onCompletion(viewModel.attributeName)
+        }
+        onCompletion(newAttributeName)
```

Comment:
I think this can be reduced to
```swift
onCompletion(viewModel.newAttributeName ?? viewModel.attributeName)
```
or rename `viewModel.attributeName` to `viewModel.originalAttributeName` and then create a new `viewModel.attributeName`(that you can unit test) that has that merge operation.
```swift
onCompletion(viewModel.attributeName)
```

Reply:
Ah, yes! I thought this approach seemed unnecessarily complicated but I forgot I could just use `??` here.

But I really like the idea of unit testing this and having `viewModel.attributeName` separate from `viewModel.originalAttributeName` for clarity, so I've gone with that suggestion. I made that change in https://github.com/woocommerce/woocommerce-ios/pull/3781/commits/4e04677777b8adf4ee0bf028266dfe999b08b7c3.